tools/syz-declextract/README.md

```diff
@@ -0,0 +1,35 @@
+# syz-declextract
+## Linux Kernel (For testing purposes)
+```
+export KERNEL=$PWD/linux-stable
+git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git $KERNEL
+cd $KERNEL
+make CC=clang defconfig # Having clang as the compiler is optional but removes erros later on
+./scripts/config -e FTRACE_SYSCALLS
+make CC=clang olddefconfig
+make CC=clang -j`nproc` # kernel has to be built at least once for the script to work
+./scripts/clang/gen_compile_commands.py
+```
+## LLVM Project
+```
+LLVM=$PWD/llvm-project
+git clone https://github.com/llvm/llvm-project.git $LLVM
```

Comment:
Also add a command that checks out the concrete llvm commit you are using now.
It's also to have a known commit on which it's known to work.
(as you've seen with the old syz-declextract that does not work with HEAD llvm and we don't know now which revision I used at the time).

Reply:
Done!